### DISCUSSION TITLE
iOS 16.x - wire:navigate and @persist

### DISCUSSION THREAD

**Discussion by davideprevosto** [livewire/livewire#8968] [7. Bugs (archived)] [answered]
### Livewire version

v3.5.12

### Laravel version

v11.29.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

Using the @persist directive is breaking the wire:navigate functionality on iOS 16.x

Please clone the repository and try to navigate using the simple `nav` element available on the /test endpoint. It is working fine on iPhone devices using iOS 17+, Android and on desktop as well, but not on a iPhone device is using iOS 16.x. 

The error shown into the browser console (I used the Safari browser simulator) is:

`Unhandled Promise Rejection: SyntaxError: The string did not match the expected pattern.`

<img width="755" alt="image" src="https://github.com/user-attachments/assets/a0e0d3a8-cda1-4f53-899d-8cb571367a9a">

I was able to navigate only removing the `@persist` directive from the `app.blade.php` layout:

```

<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        <title>{{ $title ?? 'Page Title' }}</title>
    </head>
    <body>
        {{ $slot }}

        @persist('player')
            <div>Persisting element</div>
        @endpersist
    </body>
</html>
```

Thank you

### Repository or Wirebox URL

https://github.com/davideprevosto/livewire-navigate-bug

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by luanfreitasdev** [comment, marked as answer]
https://github.com/livewire/livewire/pull/8972
